src/Mvc/Mvc.Core/src/PublicAPI.Shipped.txt

```diff
@@ -1139,6 +1139,12 @@ Microsoft.AspNetCore.Mvc.ModelBinding.Binders.ComplexObjectModelBinderProvider.G
 Microsoft.AspNetCore.Mvc.ModelBinding.Binders.ComplexTypeModelBinder
 Microsoft.AspNetCore.Mvc.ModelBinding.Binders.ComplexTypeModelBinderProvider
 Microsoft.AspNetCore.Mvc.ModelBinding.Binders.ComplexTypeModelBinderProvider.ComplexTypeModelBinderProvider() -> void
+Microsoft.AspNetCore.Mvc.ModelBinding.Binders.DateOnlyModelBinder
```

Comment:
Do not modify this file, use the unshipped file. Additionally, if you're adding new API make sure there is an issue open to review the API. https://github.com/dotnet/aspnetcore/blob/main/docs/APIReviewProcess.md